apps/src/templates/rubrics/LearningGoals.jsx

```diff
@@ -0,0 +1,392 @@
+import React, {useEffect, useState, useRef} from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import classnames from 'classnames';
+import style from './rubrics.module.scss';
+import {
+  learningGoalShape,
+  reportingDataShape,
+  studentLevelInfoShape,
+  submittedEvaluationShape,
+  aiEvaluationShape,
+} from './rubricShapes';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import {
+  BodyThreeText,
+  BodyFourText,
+  ExtraStrongText,
+  StrongText,
+  Heading6,
+} from '@cdo/apps/componentLibrary/typography';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import EvidenceLevels from './EvidenceLevels';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import AiAssessment from './AiAssessment';
+import HttpClient from '@cdo/apps/util/HttpClient';
+import {UNDERSTANDING_LEVEL_STRINGS} from './rubricHelpers';
+
+const invalidUnderstanding = -1;
```

Comment:
per the [style guide](https://github.com/code-dot-org/code-dot-org/blob/55a56e9179fd04af587639de5e8d083d86fa64bd/STYLEGUIDE.md#javascript), please 

>Use `NAMES_LIKE_THIS` for constant values.